prodigy-columns.el

```diff
@@ -0,0 +1,102 @@
+;;; prodigy.el --- Manage external services from within Emacs -*- lexical-binding: t; -*-
```

Comment:
This is wrong

Reply:
Copy paste mistake, my bad. Added 7c9e9e4e5d53587ded0500ac0156f8936b714f13